_posts/2020-08-20-codestarts-preview.adoc

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: 'Codestarts - Preview the new way to start coding with Quarkus'
+date: 2020-08-20
+tags: extension codestart quickstart
+synopsis: The idea is simple, you pick your extensions and they are able to provide example code to get you started with it :)
+author: adamevin
+---
+
+= Preview the new way to start coding with Quarkus
+
+*Have you ever generated a Quarkus project using the Maven plugin or code.quarkus.io?*
+
+If you did, then you might have wondered why, with the breadth of Quarkus extensions,
+we would only let you generate a REST API example application. Of course,
+you can pick extensions which are going to be in the dependencies of your app,
+this is pretty cool because some extensions are providing new features without any extra code. However,
+some extensions also require configuration and/or code to work, making it pretty limited as it is.
+
+== Introducing our new code generation system
+
+This is why I am thrilled to introduce the preview of our new code generation system in Quarkus `1.7.0.Final`.
```

Comment:
I usually use the `we` rather than the `I`.

Reply:
Changed..